course/theme.json

```diff
@@ -145,6 +145,7 @@
 			"commentGapSmall" : "10px",
 			"commentGapMedium" : "1.25rem",
 			"courseNewsletterGap": "clamp(2.5rem, -0.292rem + 9.306vw, 6.688rem)",
+			"courseThemeVariation": "default",
```

Comment:
I think we could call this `variation` since we know we're already using the Course theme. 🙂 Unless it has some special meaning in `theme.json` (but I don't think so)?